rpcservers/invoices/invoices_server.go

```diff
@@ -312,14 +318,15 @@ func (s *Server) CancelInvoice(ctx context.Context,
 
 	log.Infof("Canceled invoice %v", paymentHash)
 
-	return &CancelInvoiceResp{}, nil
+	return &invoicesrpc.CancelInvoiceResp{}, nil
 }
 
 // AddHoldInvoice attempts to add a new hold invoice to the invoice database.
 // Any duplicated invoices are rejected, therefore all invoices *must* have a
 // unique payment hash.
 func (s *Server) AddHoldInvoice(ctx context.Context,
-	invoice *AddHoldInvoiceRequest) (*AddHoldInvoiceResp, error) {
+	invoice *invoicesrpc.AddHoldInvoiceRequest,
+	) (*invoicesrpc.AddHoldInvoiceResp, error) {
```

Comment:
Should break before the opening parenthesis (right before `*invoicesrpc.AddHoldInvoiceResp`) to avoid the formatter wanting to have the closing parenthesis on L329 being non-indented (which the linter complains about btw.).